yarn-project/aztec-node/src/aztec-node/server.ts

```diff
@@ -482,7 +482,7 @@ export class AztecNodeService implements AztecNode, AztecNodeAdmin, Traceable {
   }
 
   /**
-   * Method to fetch the current block number.
+   * Method to fetch the latest block number synchronized by the node.
```

Comment:
The current terminology felt confusing and TBH it's quite misleading so I helped myself with updating this comment.